src/themes/components/carousel.ts

```diff
@@ -128,6 +128,9 @@ const Carousel: ComponentStyleConfig = {
       color: 'white',
       visibility: 'hidden',
       _groupHover: { md: { visibility: 'visible' } },
+      _focus: {
+        outline: 'none',
+      },
```

Comment:
Not sure it's the best regarding accessibility, let me know what you think

Reply:
I am wondering if this change should be as part of the `container` instead of the `buttonContainer`

Reply:
Thank you for the review Virginia. I tried and it does not change anything because the issue is related to the buttons

Reply:
mmmm interesting so then I think, it looks good to me. Thank you 🌸